client/css/style.css

```diff
@@ -762,15 +762,15 @@ button {
 }
 
 #chat .messages {
-	display: table;
-	table-layout: fixed;
-	width: 100%;
 	padding: 10px 0;
 }
 
 #chat .msg {
 	word-wrap: break-word;
 	word-break: break-word; /* Webkit-specific */
+	display: flex;
```

Comment:
@Gilles123 found out that these still need prefixing for now: https://github.com/thelounge/lounge/issues/624#issuecomment-248477595
Mind adding them to this PR please? :)